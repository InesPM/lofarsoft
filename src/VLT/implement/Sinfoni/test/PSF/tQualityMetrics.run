#!/bin/sh
export PSF_DATA=$SINFONI_ROOT/data
\ln -s $PSF_DATA/target3_1.dat .
$LOFAR_CHECKTOOL tQualityMetrics
STATUS=$?
\rm -f target3_1.dat
exit $STATUS
