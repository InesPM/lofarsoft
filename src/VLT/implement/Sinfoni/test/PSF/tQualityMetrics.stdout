
Number of samples used to reconstruct the PSF:      3018
Reconstructed Strehl ratio:                    0.43724351
Estimated r0 (m) at 500 nm:                    0.12326536 m
Estimated uncorrected phase at 500 nm:         936.87673 rad^2
Estimated WFS measurement error:               0.027837714 rad^2
Estimated aliasing error at 500 nm:            4.3827265 rad^2
Estimated residual phase:                      0.23968101 rad^2
RTC estimated residual phase:                  0.041138223 rad^2


